composer.json

```diff
@@ -61,7 +61,7 @@
 		"phpunit/phpunit" : "^7.1",
 		"symfony/test-pack": "^1.0",
 		"squizlabs/php_codesniffer": "3.4",
-		"rector/rector": "dev-master"
+		"rector/rector": "^0.8"
```

Comment:
i fixed this to 0.8.56. As this is a dev tool I prefer it to not auto update on each build.

Reply:
(already on develop branch)